core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -95,6 +98,8 @@ export class OpportunitiesListComponent {
 
   ngOnInit(): void {
     this.loadingOpportunityData = true;
+    this.activePageNumber = 1;
+    this.lastPageNumber = 1000;
```

Comment:
Where do these constants come from? The first one makes sense, but the 1000 might need some explanation.